stader-cli/node/repay-exess-sd.go

```diff
@@ -0,0 +1,88 @@
+package node
+
+import (
+	"fmt"
+	"math/big"
+	"strconv"
+
+	"github.com/urfave/cli"
+
+	"github.com/stader-labs/stader-node/shared/services/gas"
+	"github.com/stader-labs/stader-node/shared/services/stader"
+	cliutils "github.com/stader-labs/stader-node/shared/utils/cli"
+	"github.com/stader-labs/stader-node/stader-lib/utils/eth"
+)
+
+func repayExcessSD(c *cli.Context) error {
+	staderClient, err := stader.NewClientFromCtx(c)
+	if err != nil {
+		return err
+	}
+	defer staderClient.Close()
+
+	// Check and assign the EC status
+	err = cliutils.CheckClientStatus(staderClient)
+	if err != nil {
+		return err
+	}
+
+	// Print what network we're on
+	err = cliutils.PrintNetwork(staderClient)
+	if err != nil {
+		return err
+	}
+
+	amountInString := c.String("amount")
```

Comment:
we should compute this amount. This should be the amount of SD above the 200% mark. We should repay that amount